src/cli_callback.c

```diff
@@ -246,25 +197,15 @@ static int upload_file(storj_env_t *env, char *bucket_id, const char *file_path,
 
     const char *file_name = get_filename_separator(file_path);
 
-    if (cli_api->dst_file == NULL) {
-        if (!file_name) {
-            file_name = file_path;
-        }
-    } else {
-        file_name = cli_api->dst_file;
+    if (!file_name) {
+        file_name = file_path;
     }
 
     // Upload opts env variables:
-    char *prepare_frame_limit = getenv("STORJ_PREPARE_FRAME_LIMIT");
-    char *push_frame_limit = getenv("STORJ_PUSH_FRAME_LIMIT");
-    char *push_shard_limit = getenv("STORJ_PUSH_SHARD_LIMIT");
-    char *rs = getenv("STORJ_REED_SOLOMON");
-
     storj_upload_opts_t upload_opts = {
-        .prepare_frame_limit = (prepare_frame_limit) ? atoi(prepare_frame_limit) : 1,
-        .push_frame_limit = (push_frame_limit) ? atoi(push_frame_limit) : 64,
-        .push_shard_limit = (push_shard_limit) ? atoi(push_shard_limit) : 64,
-        .rs = (!rs) ? true : (strcmp(rs, "false") == 0) ? false : true,
+        // NB: about +500 years from time of writing
+        .expires = 17329017831,
```

Comment:
I think this should be a little more dynamically produced ideally. Is there a way we can get the current time and add some static expiration window to it (like 5 years?)

Reply:
haha - indeed... I wasn't sure what kind of default we would want so I just copied this from the libuplink test. If you think 5 years is a reasonable time, I will set it to that.